src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceProviderOptions.cs

```diff
@@ -19,8 +19,11 @@ public class ServiceProviderOptions
         public bool ValidateScopes { get; set; }
 
         /// <summary>
-        /// Gets or sets a value that indicates whether validation is performed to ensure all services can be created when <see cref="M:BuildServiceProvider(IServiceCollection,ServiceProviderOptions)" /> is called.
+        /// Gets or sets a value that indicates whether validation is performed to ensure all services can be created when <see cref="BuildServiceProvider(IServiceCollection, ServiceProviderOptions)" /> is called.
```

Comment:
@carlossanlop I'm not sure what's wrong with this cref. Can you help?

Reply:
I think the two xml references to `BuildServiceProvider` in this file should be pointing to the fullname of the method defined in the `Microsoft.Extensions.DependencyInjection.ServiceCollectionContainerBuilderExtensions` type. It's under the same folder as ServiceProviderOptions.cs.

cc @dotnet/area-extensions-dependencyinjection

Reply:
```suggestion
        /// Gets or sets a value that indicates whether validation is performed to ensure all services can be created when <see cref="Microsoft.Extensions.DependencyInjection.ServiceCollectionContainerBuilderExtensions.BuildServiceProvider(IServiceCollection, ServiceProviderOptions)" /> is called.
```